ios/Podfile

```diff
@@ -1,3 +1,5 @@
+source 'https://cdn.cocoapods.org/'
```

Comment:
what does this do?

Reply:
It switches us to the [faster new specs repo](http://blog.cocoapods.org/CocoaPods-1.7.2/) for CocoaPods. No more need to run `pod repo update` etc.

Reply:
Nice!